### PR TITLE
SOF-7178: CLI job workflow for DeePMD molecular dynamics calculation

### PR DESCRIPTION
**Pull request by pranabdas**

### CODE REVIEW

File: deepmd/data_generation/convert.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import dpdata
+import numpy as np
+
+data = dpdata.LabeledSystem("h2o_32", fmt="qe/cp/traj")
```
Comment (timurbazhirov) [inline deepmd/data_generation/convert.py:4]: Could we add links to the documentation that explain how the conversion should be done as a comment?

File: deepmd/README.md [hunk 2]
```diff
@@ -1,23 +1,62 @@
...
+with a small amount of data for a short period of time. Consequently, the
+accuracy of the resulting model is lower than we should expect for a model
+trained for longer on a larger amount of data.
 
-### Training
 
-The training script trains a model on the water data provided in the package examples [2] and then freezes that model for production. The job must be submitted from the same directory as the job.pbs file.
+### Data Generation
+
+This example invokes a python script to transform the output of a Car-Parrinello
```
Comment (timurbazhirov) [inline deepmd/README.md:20]: Let's add a link to the CP.X folder below?

Comment (timurbazhirov) [reply deepmd/README.md:20]: And explain how to invoke it here too

File: espresso/CP/h2o_32.in [hunk 3]
```diff
@@ -0,0 +1,168 @@
+&CONTROL
```
Comment (timurbazhirov) [inline espresso/CP/h2o_32.in:1]: Maybe let's use lowercase `pw.x` and `cp.x` as names?

File: deepmd/molecular_dynamics/job.pbs [hunk 4]
```diff
@@ -15,25 +14,26 @@
...
+##PBS -q comp-002-zone-a-production-20160630-cluster-007.exabyte.io
 #PBS -l nodes=1
-#PBS -l ppn=1
-#PBS -l walltime=00:00:10:00
-#PBS -q D
-#PBS -m abe
-#PBS -M info@exabyte.io
+#PBS -l ppn=4
+#PBS -l walltime=00:01:00:00
+#PBS -q OR
```
Comment (timurbazhirov) [inline deepmd/molecular_dynamics/job.pbs:26]: For the queues - let's keep "D" as the calculation does fit there and the wait is shorter

File: deepmd/model_training/se_e2_r_input.json [hunk 5]
```diff
@@ -42,14 +42,13 @@
     "_comment": " traing controls",
     "training" : {
 	"training_data": {
-	    "systems":		["../data/data_0/", "../data/data_1/", "../data/data_2/"],
+	    "systems":		["../data/training_data/"],
```
Comment (timurbazhirov) [inline deepmd/model_training/se_e2_r_input.json:45]: 👍

File: espresso/CP/h2o_32.in [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+  title = "water 32 Molecules - PBE 330K",
+  calculation = 'cp-wf',
+  restart_mode = "from_scratch",
+  prefix = "h2o_32",
+  pseudo_dir = '.',
+  outdir = '.',
+  nstep = 1000,
```
Comment (timurbazhirov) [inline espresso/CP/h2o_32.in:8]: It takes quite a while to run this - could we reduce the number of steps for the purpose of this example? Perhaps we can also explain in README that this is not a "production" run, but just a calculation for demonstration purposes.

Comment (timurbazhirov) [reply espresso/CP/h2o_32.in:8]: 100 stop would suffice

File: deepmd/model_training/se_e2_r_input.json (hunk 5 above)
Comment (timurbazhirov) [reply deepmd/model_training/se_e2_r_input.json:45]: Perhaps we could omit `_data` at the end here and below

File: deepmd/README.md [hunk 7]
```diff
@@ -1,23 +1,62 @@
 # DEEPMD
 
-DeePMD (Deep Potential Molecular Dynamics) [1] is a package for easily building performant neural network potentials for carrying out molecular dynamics. The DeePMD code base has been developed such that trained models can be interfaced directly to LAMMPS.
+DeePMD (Deep Potential Molecular Dynamics) [1] is a package for easily building
```
Comment (timurbazhirov) [inline deepmd/README.md:3]: I think we need a script that would tie everything up together, starting from cp.x and finishing with the MD run

File: deepmd/README.md [hunk 8]
```diff
@@ -1,23 +1,62 @@
...
+The molecular dynamics script takes the `graph.pb` file produced by the training
+example and carries out a prospective simulation using LAMMPS. As above the job
+must be submitted from the same directory as the `molecular_dynamics/job.pbs`
+file.
+
+
+### Model inference
+
+DP model can be used to evaluate the energy of each frame in the LAMMPS
+trajectory.
```
Comment (timurbazhirov) [inline deepmd/README.md:49]: We should elaborate what this is for and maybe add a script for this, similar to `convert.py`

File: deepmd/molecular_dynamics/job.pbs (hunk 4 above)
Comment (pranabdas) [reply deepmd/molecular_dynamics/job.pbs]: Some tasks (e.g., `model_training/job.pbs`) hit the memory limit of the debug queue.

**Comment by timurbazhirov** [comment]
Superseded by https://github.com/Exabyte-io/cli-job-examples/pull/13/files

**Pull request closed without merging**